Known_Issues/All VMMS operations are blocked on Silver or higher durability.md

```diff
@@ -0,0 +1,100 @@
+# VMMS operations are blocked node gets stuck in deletion and creation after deleting a VMSS
+
+## Problem
+- All VMSS operations are blocked on Silver or higher durability Scaleset after trying to delete a node from a VMMS 
```

Comment:
I would suggest linking to the public docs that talk about what durability is https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-capacity#durability-characteristics-of-the-cluster 

suggestion "silver or higher durability Service Fabric VMs..."

typo at end VMMS - > VMSS

Reply:
fixed